<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Volker Krause &lt;vkrause@kde.org&gt;, KovoKs &lt;tomalbers@kde.nl&gt;</author>
 <class>SMTPSettings</class>
 <widget class="QWidget" name="SMTPSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>808</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>855</width>
        <height>808</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Account Information</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="hostLabel">
            <property name="text">
             <string>Mail &amp;server:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_host</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KLineEdit" name="kcfg_host"/>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="kcfg_requiresAuthentication">
            <property name="text">
             <string>Server &amp;requires authentication</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="usernameLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Login:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_userName</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="KLineEdit" name="kcfg_userName">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="passwordLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>P&amp;assword:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>password</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="password">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>The password to send to the server for authorization.</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="kcfg_storePassword">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Store SMTP password</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="KButtonGroup" name="kcfg_encryption">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="title">
          <string>Connection Settings</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QStackedWidget" name="checkCapabilitiesStack">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="page">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="KPushButton" name="checkCapabilities">
                <property name="text">
                 <string>Auto Detect</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="page_2">
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QProgressBar" name="checkCapabilitiesProgress">
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="noAuthPossible">
            <property name="visible">
             <bool>false</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>0</number>
            </property>
            <property name="text">
             <string>This server does not support authentication</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Encryption:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="encryptHLayout">
              <item>
               <widget class="QRadioButton" name="none">
                <property name="text">
                 <string>&amp;None</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="ssl">
                <property name="text">
                 <string>&amp;SSL</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="tls">
                <property name="text">
                 <string>&amp;TLS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="portLabel">
              <property name="text">
               <string>&amp;Port:</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>kcfg_port</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KIntNumInput" name="kcfg_port">
              <property name="value">
               <number>25</number>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="authLabel">
              <property name="text">
               <string>Authentication:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KComboBox" name="authCombo"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="smtpSettingsGroupBox">
         <property name="title">
          <string>SMTP Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="kcfg_specifyHostname">
            <property name="text">
             <string>Sen&amp;d custom hostname to server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hostnameLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Hos&amp;tname:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buddy">
             <cstring>kcfg_localHostname</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KLineEdit" name="kcfg_localHostname">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="kcfg_specifySenderOverwriteAddress">
            <property name="text">
             <string>Use custom sender address</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Sender Address:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="KLineEdit" name="kcfg_senderOverwriteAddress">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="precommandLabel">
            <property name="text">
             <string>Precommand:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="KLineEdit" name="kcfg_precommand"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>hostnameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifyHostname</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_localHostname</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_userName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>usernameLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>128</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>password</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_requiresAuthentication</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_storePassword</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifySenderOverwriteAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_specifySenderOverwriteAddress</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_senderOverwriteAddress</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
